src/app/views/query-runner/QueryRunner.tsx

```diff
@@ -43,8 +44,8 @@ export class QueryRunner extends Component<
 
   private handleOnUrlChange = (newUrl = '') => {
     this.setState({ url: newUrl });
-
     this.changeUrlVersion(newUrl);
+    this.initialiseAutoComplete(newUrl);
```

Comment:
This is misleading, it implies that we are initialising auto complete whenever the url changes. I suggest that we move the check we do to `handleOnUrlChange`.

```js
if (newUrlEndsWith?) {
  this.initialiseAutoComplete()
}
```